### PR TITLE
fixes cmd_att test and some more

### PR DESCRIPTION
**Pull request by rlaemmert**

### CODE REVIEW

File: libr/anal/fcn.c [hunk 1]
```diff
@@ -880,8 +880,8 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 						ls_free (secs);
 					}
 					if (s) {
-						must_eob = (s->vaddr <= op.jump);
-						must_eob &= ((s->vaddr + s->vsize) > op.jump);
+						must_eob = (s->vaddr > op.jump);
+						must_eob |= ((s->vaddr + s->vsize) < op.jump);
```
Comment (MaskRay) [inline libr/anal/fcn.c]: `<=`

There is a regression introduced at https://github.com/radare/radare2/commit/0161fa0ec2d1f20f38873d6a48ae13e08e2cc138#diff-49a52329a1b3f5e5f62d571577391074L880

Comment (rlaemmert) [inline libr/anal/fcn.c]: so that the last possible address cannot be the target of the jump?

**Comment by radare** [comment]
please resolve conflicts

**Comment by alvarofe** [comment]
Fixed in master

**Comment by MaskRay** [comment]
No. you missed an `=`

`s?` this condition also looks suspicious to me.

**Comment by alvarofe** [comment]
Yep, i made a mistake there i pushed yesterday the changes after noticing it

**Pull request closed without merging**